browser/ui/brave_browser_window.cc

```diff
@@ -46,3 +46,8 @@ bool BraveBrowserWindow::ShowBraveHelpBubbleView(const std::string& text) {
 bool BraveBrowserWindow::IsInTabDragging() const {
   return false;
 }
+
+bool BraveBrowserWindow::PreHandleMouseEvent(
```

Comment:
what upstream behavior are we overriding here?

Reply:
cc: @simonhong

Reply:
Filed - https://github.com/brave/brave-browser/issues/49430